### ISSUE TITLE
Avoid org.jdom.IllegalDataException on adding console output

### ISSUE THREAD

**Issue by vlad2011** [integrity-tf/integrity#178]
Console output must be checked and sanitized to avoid exceptions like following:
```
org.jdom.IllegalDataException: The data "WILDFLY START (wildfly-1): [0m20:50:25,775 INFO  [org.jboss.modules] (main) JBoss Modules version 1.5.2.Final" is not legal for a JDOM attribute: 0x1b is not a legal XML character.
	at org.jdom.Attribute.setValue(Attribute.java:491)
	at org.jdom.Attribute.<init>(Attribute.java:228)
	at org.jdom.Attribute.<init>(Attribute.java:251)
	at org.jdom.Element.setAttribute(Element.java:1128)
	at de.gebit.integrity.runner.callbacks.xml.XmlWriterTestCallback.addConsoleOutput(XmlWriterTestCallback.java:2297)
	at de.gebit.integrity.runner.callbacks.xml.XmlWriterTestCallback.internalOnCallFinish(XmlWriterTestCallback.java:1451)
	at de.gebit.integrity.runner.callbacks.xml.XmlWriterTestCallback.onCallFinish(XmlWriterTestCallback.java:1437)
	at de.gebit.integrity.runner.callbacks.CompoundTestRunnerCallback.onCallFinish(CompoundTestRunnerCallback.java:189)
	at de.gebit.integrity.runner.callbacks.CompoundTestRunnerCallback.onCallFinish(CompoundTestRunnerCallback.java:189)
	at de.gebit.integrity.runner.DefaultTestRunner.executeCallSingle(DefaultTestRunner.java:2349) 
```

**Comment by profhenry** [comment]
Duplicate of #156, already fixed

**Comment by S1artie** [comment]
Indeed, thanks for commenting. This was already fixed in 0.16.6 and 0.17.0.
